docs/source/resources/transition_to_ft_v1.0.ipynb

```diff
@@ -0,0 +1,822 @@
+    "\n",
+    "Starting in Featuretools 1.0, column type inference is being handled by Woodwork. Any columns that do not have a logical type assigned by the user when adding a dataframe to an EntitySet will have their logical types inferred by Woodwork. As before, type inference can be skipped for any columns in a dataframe by passing the appropriate logical types in a dictionary when calling `EntitySet.add_dataframe`.\n",
+    "\n",
+    "As an example, we can create a new dataframe and add it to an EntitySet, specifying the logical type for the users full name as the Woodwork `PersonFullName` logical type."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a34016b5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "users_df = pd.DataFrame({\n",
+    "    'id': [0, 1, 2],\n",
+    "    'name': ['John Doe', 'Rita Book', 'Teri Dactyl']\n",
+    "})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d999e022",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "es.add_dataframe(dataframe=users_df,\n",
+    "                 dataframe_name='users',\n",
+    "                 index='id',\n",
+    "                 logical_types={'name': 'PersonFullName'})\n",
+    "\n",
+    "es['users'].ww"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d2eff5e1",
+   "metadata": {},
+   "source": [
+    "Looking at the typing information above, we can see that the logical type for the `name` column was set to `PersonFullName` as we specified.\n",
+    "\n",
+    "Situations will occur where type inference identifies a column as having the incorrect logical type. In these situations, the logical type can be updated using the Woodwork `set_types` method. Let's say we want the `order_id` column of the `orders` dataframe to have a `Categorical` logical type instead of the `Integer` type that was inferred. Previously, this would have accomplished through the `Entity.convert_variable_type` method.\n",
+    "\n",
+    "```python\n",
+    "from featuretools.variable_types import Categorical\n",
+    "\n",
+    "es['items'].convert_variable_type(variable_id='order_id', new_type=Categorical)\n",
+    "```\n",
+    "\n",
+    "Now, we can perform this same update using Woodwork:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a6c095b5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "es['items'].ww.set_types(logical_types={'order_id': 'Categorical'})\n",
+    "es['items'].ww"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ebcd6d9e",
+   "metadata": {},
+   "source": [
+    "### Mapping from old Featuretools variable types to Woodwork ColumnSchemas\n",
```

Comment:
Since column schemas are not passed in in this way at woodwork init, we might need to spend a little time explaining how these column schemas can be achieved in their EntitySets since I'm not sure how obvious the connection between ColumnSchemas and init parameters are.

I think linking to the section of the woodwork in featuretools guide on ColumnSchemas might be helpful (/notebooks/docs/source/getting_started/woodwork_types.ipynb#Woodwork-in-DFS)

Reply:
Like, the table below is going to be really useful for determining input and return types for Primitives, but I'm not sure it covers `init` parameters (since, for example, adding the `numeric` tag at woodwork init isn't super useful when it comes to actually avoiding type inference). Not totally sure what the fix would be. Adding another column for "init parameters" seems overkill since many will be essentially the same. Maybe we only add additional info when it's necessary in a "additional info" column